packs/data/strength-of-thousands-bestiary.db/ixamè.json

```diff
@@ -350,7 +350,7 @@
                 "attackEffects": {
                     "custom": "",
                     "value": [
-                        "1d12-poison"
+                        "1d12 poison"
```

Comment:
Wondering if we did a PDF to Foundry actor import for SoT and it was tossing these in there.